client/src/context/useKanbanContext.tsx

```diff
@@ -1,45 +1,84 @@
-import { useState, useContext, createContext, FunctionComponent } from 'react';
+import { useState, useContext, createContext, FunctionComponent, useEffect } from 'react';
 import { DraggableLocation, DropResult } from 'react-beautiful-dnd';
-import { columnData } from '../components/Kanban/data';
+import cloneDeep from 'lodash.clonedeep';
+import { getUserBoards, updateBoard } from '../helpers/';
+import { useSnackBar } from './useSnackbarContext';
 import { IKanbanContext } from '../interface/KanbanContext';
 import { IColumn } from '../interface/Column';
 import { ICard } from '../interface/Card';
-import cloneDeep from 'lodash.clonedeep';
-import { useSnackBar } from './useSnackbarContext';
+import { IBoard } from '../interface/Board';
+import { useAuth } from './useAuthContext';
 
 export const KanbanContext = createContext<IKanbanContext>({} as IKanbanContext);
 
 export const KanbanProvider: FunctionComponent = ({ children }): JSX.Element => {
-  const [columns, setColumns] = useState<Array<IColumn>>(columnData);
+  const [activeBoard, setActiveBoard] = useState<IBoard>({
+    _id: 'Initial',
+    name: 'Initial',
+    columns: [],
+    cards: [],
+    user: 'Initial',
+    createdAt: 'Initial',
+  });
+  const [columns, setColumns] = useState<IColumn[]>(activeBoard.columns);
```

Comment:
Changed the way `columns` are used and incorporated app's usage for an `activeBoard`. App now has one board that remains active at all times and can be changed accordingly